pyrobosim/examples/demo_pddl.py

```diff
@@ -13,6 +13,10 @@
 from pyrobosim.gui import start_gui
 from pyrobosim.planning.pddlstream import PDDLStreamPlanner, get_default_domains_folder
 from pyrobosim.utils.general import get_data_folder
+from pddlstream.language.constants import Exists, And
+
+PARAM1 = "?o"
+PARAM2 = "?o"
```

Comment:
I would probably prefer you just used the literal `"?o"` directly in the expression below

... and actually, it should be `?l` or `?loc` since the `Has` derived predicate accepts `Location` type and not `Object`.